site/docs/v1/tech/apis/_import-users-request-body.adoc

```diff
@@ -103,7 +103,7 @@ The User's mobile phone number. This is useful is you will be sending push notif
 [field]#users``[x]``.password# [type]#[String]# [required]#Required#::
 The User's password. If [field]#encryptionScheme# is provided this value is the existing password hash, if [field]#encryptionScheme# is not provided this value is assumed to be plain text and it will be hashed during import.
 +
-[field]#password# should contain only the password hash.
+The value of [field]#password# should contain only the password hash, or, if the [field]#encryptionScheme# is omitted, the plain text password.
 Unless you are using a custom plugin that does not utilize base64, [field]#password# should be base64 encoded.
 It is recommended to use a base64 encoded string, but if you are importing hashes from a legacy system that uses base16, base32, or another encoding and are writing a custom plugin, import the hash in the same format the plugin produces.
```

Comment:
Is all of this one line, or do we need line breaks? I don't think this will render as new lines in the doc will it? (unless you add a new line with `+`?)